components/checkbox/dist/metadata.json

```diff
@@ -2,59 +2,49 @@
+    ".spectrum-Checkbox .spectrum-Checkbox-input:disabled",
     ".spectrum-Checkbox .spectrum-Checkbox-input:disabled + .spectrum-Checkbox-box:before",
     ".spectrum-Checkbox .spectrum-Checkbox-input:disabled ~ .spectrum-Checkbox-label",
+    ".spectrum-Checkbox .spectrum-Checkbox-input:focus-visible + .spectrum-Checkbox-box:after",
+    ".spectrum-Checkbox .spectrum-Checkbox-input:focus-visible + .spectrum-Checkbox-box:before",
+    ".spectrum-Checkbox .spectrum-Checkbox-input:focus-visible + .spectrum-Checkbox-label",
+    ".spectrum-Checkbox .spectrum-Checkbox-label",
     ".spectrum-Checkbox .spectrum-Checkbox-partialCheckmark",
     ".spectrum-Checkbox--emphasized .spectrum-Checkbox-input:checked + .spectrum-Checkbox-box:before",
-    ".spectrum-Checkbox--emphasized .spectrum-Checkbox-input:focus-visible:checked + .spectrum-Checkbox-box:before",
     ".spectrum-Checkbox--emphasized.is-indeterminate .spectrum-Checkbox-box:before",
+    ".spectrum-Checkbox--emphasized.is-indeterminate .spectrum-Checkbox-input.is-focus-visible + .spectrum-Checkbox-box:before",
     ".spectrum-Checkbox--emphasized.is-indeterminate .spectrum-Checkbox-input:checked + .spectrum-Checkbox-box:before",
-    ".spectrum-Checkbox--emphasized.is-indeterminate .spectrum-Checkbox-input:focus-visible + .spectrum-Checkbox-box:before",
     ".spectrum-Checkbox--emphasized.is-indeterminate:active .spectrum-Checkbox-box:before",
     ".spectrum-Checkbox--emphasized.is-indeterminate:active .spectrum-Checkbox-input:checked + .spectrum-Checkbox-box:before",
     ".spectrum-Checkbox--emphasized.is-indeterminate:hover .spectrum-Checkbox-box:before",
     ".spectrum-Checkbox--emphasized.is-indeterminate:hover .spectrum-Checkbox-input:checked + .spectrum-Checkbox-box:before",
-    ".spectrum-Checkbox--emphasized.is-invalid .spectrum-Checkbox-input:focus-visible:checked + .spectrum-Checkbox-box:before",
-    ".spectrum-Checkbox--emphasized.is-invalid.is-indeterminate:hover .spectrum-Checkbox-box:before",
-    ".spectrum-Checkbox--emphasized.is-invalid.is-indeterminate:hover .spectrum-Checkbox-input:checked + .spectrum-Checkbox-box:before",
-    ".spectrum-Checkbox--emphasized.is-invalid:active .spectrum-Checkbox-box:before",
-    ".spectrum-Checkbox--emphasized.is-invalid:active .spectrum-Checkbox-input:checked + .spectrum-Checkbox-box:before",
-    ".spectrum-Checkbox--emphasized.is-invalid:hover .spectrum-Checkbox-input:checked + .spectrum-Checkbox-box:before",
+    ".spectrum-Checkbox--emphasized.is-indeterminate:not(.is-disabled):focus-visible .spectrum-Checkbox-box:before",
+    ".spectrum-Checkbox--emphasized.is-indeterminate:not(.is-disabled):focus-visible .spectrum-Checkbox-input:checked + .spectrum-Checkbox-box:before",
+    ".spectrum-Checkbox--emphasized.is-readOnly:hover .spectrum-Checkbox-input:checked + .spectrum-Checkbox-box:before",
+    ".spectrum-Checkbox--emphasized.spectrum-Checkbox-input:not(.is-disabled):checked:focus-visible + .spectrum-Checkbox-box:before",
```

Comment:
Correct me if I'm wrong, but it looks like `.is-disabled` may be a similar situation to `.is-focus-visible` mentioned in the previous comments. I'm not seeing any mention of this class in the original metadata.json, or in the docs about implementations needing to apply it. I think any use of this class needs to be adjusted to use the existing `:disabled` selector with the input.